docs/source/user-guide/tasks/work-with-packages.rst

```diff
@@ -113,6 +113,11 @@ Package Channels
   *Breaking Change in version 0.11.0*. All channels you wish to search through for packages must be supplied on the CLI
   or in the project YAML file.  To support reproducible projects that build the same way for different users, Anaconda Project will not respect channels declared in your ``.condarc`` file.
 
+.. note::
+  *Backwards compatibility fix in version 0.11.1*. The ``defaults`` channel is always included when packages are installed
```

Comment:
To be more explicit?

```suggestion
  *Backwards compatibility fix in version 0.11.1*. The ``defaults`` channel is always appended when packages are installed
```

Reply:
Given the latest discussion, it sounds like there should also be a sentence along the lines of:

'If the defaults channel is explicitly specified in the channel list, the stated channel ordering will be used and the defaults channel will not be appended again at the end of the list.'